codacy/values-production.yaml

```diff
@@ -365,7 +365,7 @@ crow:
       http:
         context: "/monitoring"
     passwordAuth:
-      password: <--- crow-login-password --->
+      password: C0dacy123
```

Comment:
Since the default password is well-known (it is listed in the documentation), it's simpler to make the default password explicit in the .yaml file. Otherwise, customers won't know what to put here until they later read the page on how to set up the monitoring.